src/Normalizer/CommentItemNormalizer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\relaxed\Normalizer\CommentItemNormalizer.
+ */
+
+namespace Drupal\relaxed\Normalizer;
+
+use Drupal\serialization\Normalizer\NormalizerBase;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+class CommentItemNormalizer extends NormalizerBase implements DenormalizerInterface {
+
+  /**
+   * The interface or class that this Normalizer supports.
+   *
+   * @var string
+   */
+  protected $supportedInterfaceOrClass = 'Drupal\comment\Plugin\Field\FieldType\CommentItem';
+
+  /**
+   * @var string[]
+   */
+  protected $format = array('json');
+
+  /**
+   * {@inheritdoc}
+   */
+  public function normalize($field, $format = NULL, array $context = array()) {
+    $value = $field->getValue();
+    $field_info = array();
+    $reference_fields = array('cid', 'last_comment_uid');
+    foreach ($value as $key => $item) {
+      if (in_array($key, $reference_fields) && is_numeric($item)) {
+        $field_info[$key] = NULL;
+      }
+      else {
+        $field_info[$key] = $item;
+      }
+    }
```

Comment:
Can we explain with some comments what's going on here and why we need to do this? And why won't a regular `EntityReferenceNormalizer` work here?